### DISCUSSION TITLE
Communication

### DISCUSSION THREAD

**Discussion by canacar7** [OpenDDS/OpenDDS#3928] [Q&A] [answered]
I developed two applications, publisher and subscriber, on Eclipse. I can communicate on the same computer. But I want to run the sub/pub side on another computer. And for this, I connect two computers using a switch. Then I ping the PCs and confirm there is communication. then when i run pub\sub applications i expect message but i don't get any message. The ini files of dds are designed to run in rtps mode. How can I fix this, thanks

**Comment by jrw972** [comment]
Turning on logging and posting the logs would be a good place to start.

**Comment by canacar7** [comment, marked as answer]
I guess when changing ipv4 address, the ipv6 address was being deleted. When I added the ipv6 address again, it enabled communication.
